src/index.js

```diff
@@ -149,6 +150,7 @@ const args = (() => {
       nogl: minimistArgs.nogl,
       headless: minimistArgs.headless,
       uncapped: minimistArgs.uncapped,
+      onBeforeLoad: minimistArgs.onbeforeload || false
```

Comment:
If it's a string as specified above, `false` is probably not a good default, and not consistent with the other arguments. I think we can leave it off.